plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/logging/LogMark.js

```diff
@@ -46,7 +43,8 @@ Ext.define('NX.coreui.view.logging.LogMark', {
         xtype: 'textfield',
         name: 'message',
         itemId: 'message',
-        fieldLabel: NX.I18n.get('ADMIN_LOG_VIEWER_MARK_MESSAGE')
+        fieldLabel: NX.I18n.get('ADMIN_LOG_VIEWER_MARK_MESSAGE'),
+        allowBlank: false
```

Comment:
if you do not allow blank  you probably also need to blur on validation to avoid validation on cancel.